compiler/luci-interpreter/src/loader/GraphLoader.cpp

```diff
@@ -162,6 +204,10 @@ void GraphLoader::loadTensors()
     {
       shape = getNodeShape(variable_node);
     }
+    else if (const auto *custom_out_node = dynamic_cast<const luci::CircleCustomOut *>(node))
+    {
+      shape = getNodeShape(custom_out_node);
+    }
```

Comment:
Since this place has not small (4 for now) number of cases with `dynamic_cast` usage I think better to rewrite this place with `switch(opcode)-case` construction in order increase performance and reduce code duplication.

Reply:
P.S. In another PR.

+cc @binarman